tests/ocpp-server/server.py

```diff
@@ -93,6 +95,11 @@ async def on_meter_values(self, evse_id: int, meter_value, **kwargs):
         logging.info("Received %s", Action.MeterValues)
         return ocpp.v201.call_result.MeterValues()
 
+    @on(Action.GetBaseReport)
+    async def on_get_base_report(self, request_id: int, report_base: ReportBaseType, **kwargs):
```

Comment:
Is it the args name matching the OCPP 2 payload spec?

Reply:
I followed below.

https://github.com/mobilityhouse/ocpp/blob/master/ocpp/v201/schemas/GetBaseReportRequest.json